extensions/procfs/tests/MemInfoTests.cpp

```diff
@@ -0,0 +1,39 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "Catch.h"
+#include "ProcFs.h"
+
+using org::apache::nifi::minifi::extensions::procfs::ProcFs;
+using org::apache::nifi::minifi::extensions::procfs::MemInfo;
+
+TEST_CASE("ProcFSTest meminfo test with mock", "[procfmeminfomocktest]") {
+  ProcFs proc_fs("./mockprocfs_t0");
+  auto mem_info = proc_fs.getMemInfo();
+  REQUIRE(mem_info);
+  CHECK(mem_info->getTotalMemory() == 32895000LL*1024);
+  CHECK(mem_info->getFreeMemory() == 9870588LL*1024);
+  CHECK(mem_info->getAvailableMemory() == 22167288LL*1024);
+  CHECK(mem_info->getTotalSwap() == 9227464LL*1024);
+  CHECK(mem_info->getFreeSwap() == 9188320LL*1024);
```

Comment:
note: we have a `Literals.h` which has a `_KiB`

Reply:
good idea, changed it in https://github.com/apache/nifi-minifi-cpp/pull/1152/commits/f06938c35f0943a66c4bd1d7b895a02da8c0a83f